driver-core/src/main/java/com/datastax/driver/core/AggregateMetadata.java

```diff
@@ -101,11 +106,29 @@ static AggregateMetadata build(KeyspaceMetadata ksm, Row row, VersionNumber vers
         if (version.getMajor() >= 3) {
             stateType = DataTypeCqlNameParser.parse(stateTypeName, cluster, ksm.getName(), ksm.userTypes, null, false, false);
             String rawInitCond = row.getString("initcond");
-            initCond = rawInitCond == null ? null : codecRegistry.codecFor(stateType).parse(rawInitCond);
+            if (rawInitCond == null) {
+                initCond = null;
+            } else {
+                try {
+                    initCond = codecRegistry.codecFor(stateType).parse(rawInitCond);
```

Comment:
It looks like it's possible this change is no longer needed, especially given @pcmanus' [comment](https://datastax-oss.atlassian.net/browse/JAVA-1046?focusedCommentId=26546&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-26546):

> Fyi, I think we should actually change INITCOND to use a blob literal for custom type values (see my comment on CASSANDRA-11064 so you might want to hold on this a bit, as you may not have to change anything.

With and without this change, the behavior is exactly the same, although the PR seems harmless.  I think at the very least we should update the test and include that.